clients/core-catalog-writer/build.gradle.kts

```diff
@@ -0,0 +1,57 @@
+plugins {
+    id("org.openapi.generator") version "7.3.0"
+}
+val currentDirectory = "$rootDir/clients/core-catalog-writer"
```

Comment:
I think gradle have set some variable like `$projectDir` that have the same meaning.